src/egon/data/metadata.py

```diff
@@ -0,0 +1,262 @@
+        "path": "https://www.gesetze-im-internet.de/geonutzv/",
+        "instruction": "Geodaten und Geodatendienste, einschließlich "
+                       "zugehöriger Metadaten, werden für alle derzeit "
+                       "bekannten sowie für alle zukünftig bekannten Zwecke "
+                       "kommerzieller und nicht kommerzieller Nutzung "
+                       "geldleistungsfrei zur Verfügung gestellt, soweit "
+                       "durch besondere Rechtsvorschrift nichts anderes "
+                       "bestimmt ist oder vertragliche oder gesetzliche "
+                       "Rechte Dritter dem nicht entgegenstehen.",
+        "attribution": attribution
+    }
+
+
+def generate_resource_fields_from_sqla_model(model):
+    """ Generate a template for the resource fields for metadata from a SQL
+    Alchemy model.
+
+    For details on the fields see field 14.6.1 of `Open Energy Metadata
+    <https://github.com/OpenEnergyPlatform/ oemetadata/blob/develop/metadata/
+    v141/metadata_key_description.md>`_ standard.
+    The fields `name` and `type` are automatically filled, the `description`
+    and `unit` must be filled manually.
+
+    Examples
+    --------
+    >>> from egon.data.metadata import generate_resource_fields_from_sqla_model
+    ... from egon.data.datasets.zensus_vg250 import Vg250Sta
+    >>> resources = generate_resource_fields_from_sqla_model(Vg250Sta)
+
+    Parameters
+    ----------
+    model : sqlalchemy.ext.declarative.declarative_base()
+        SQLA model
+
+    Returns
+    -------
+    list of dict
+        Resource fields
+    """
+
+    return [{'name': col.name,
+             'description': '',
+             'type': str(col.type).lower(),
+             'unit': 'none'}
+            for col in model.__table__.columns]
+
+
+def generate_resource_fields_from_db_table(schema, table):
+    """ Generate a template for the resource fields for metadata from a
+    database table.
+
+    For details on the fields see field 14.6.1 of `Open Energy Metadata
+    <https://github.com/OpenEnergyPlatform/ oemetadata/blob/develop/metadata/
+    v141/metadata_key_description.md>`_ standard.
+    The fields `name` and `type` are automatically filled, the `description`
+    and `unit` must be filled manually.
+
+    Examples
+    --------
+    >>> from egon.data.metadata import generate_resource_fields_from_db_table
+    >>> resources = generate_resource_fields_from_db_table('openstreetmap', 'osm_point')
+
+    Parameters
+    ----------
+    schema : str
+        The target table's database schema
+    table : str
+        Database table on which to put the given comment
+
+    Returns
+    -------
+    list of dict
+        Resource fields
+    """
+
+    # TODO: Workaround as SQLA does not recognize Geometry columns. This only
+    #  covers the dtype of a specific column 'geom', how can this be done in a
+    #  generic way?
+    ischema_names['geom'] = Geometry
```

Comment:
As you already explained in the meeting last week this might be a bit problematic for SQL-defined tables with another geom column name. Do you want to find a solution (e.g. giving the geom-column names as an input-parameter to the function) before this is merged or prefer to solve it when we run into problems?

Reply:
I'd implement the geom-columns-as-input-params as minimum solution.
I haven't put more efforts into a more generic approach, what about u @gnn?

Reply:
> I'd implement the geom-columns-as-input-params as minimum solution.

Done in ec7d5eefdd2bc4eae21bb68135da23d527aa3bfc